supervisor/http_client.py

```diff
@@ -201,7 +250,10 @@ def chunked_size(self):
         line = self.buffer
         if not line:
             return
-        chunk_size = int(line.split()[0], 16)
+        try:
+            chunk_size = int(line.split()[0], 16)
```

Comment:
Silently swallowing an exception always makes me nervous. What's the purpose here? Maybe add a comment?